iot_packet_verifier/src/balances.rs

```diff
@@ -67,13 +67,13 @@ where
 {
     type Error = S::Error;
 
-    /// Debits the balance from the cache, returning true if there was enough
-    /// balance and false otherwise.
+    /// Debits the balance from the cache, returning the remaining balance if there
+    /// was enough balance and false otherwise.
```

Comment:
```suggestion
    /// Debits the balance from the cache, returning the remaining balance as an
    /// option if there was enough and none otherwise.
```